fuse/data/pipelines/pipeline_default.py

```diff
@@ -54,6 +56,12 @@ def __init__(
             self._op_ids = op_ids
         self._verbose = verbose
 
+    def copy(self):
```

Comment:
Nice,
Can you please add in a comment that it is a shallow copy (both pipelines will still use the same instances of ops)?

Reply:
done